recipes-samples/packagegroups/packagegroup-rpb.bb

```diff
@@ -2,16 +2,13 @@ SUMMARY = "Organize packages to avoid duplication across all images"
 
 inherit packagegroup
 
-DOCKER = "docker"
-DOCKER_armv7a = ""
-
 # contains basic dependencies, that can work without graphics/display
 RDEPENDS_packagegroup-rpb = "\
     96boards-tools \
     alsa-utils-aplay \
     coreutils \
     cpufrequtils \
-    ${DOCKER} \
+    ${@bb.utils.contains("TARGET_ARCH", "arm", "", "docker", d)} \
```

Comment:
it's because we're building for v7a and aarch64 only. It isn't equivalent, now it filters out also v4/v5. But we don't build those.